mempool/clist_mempool.go

```diff
@@ -662,6 +662,39 @@ func (mem *CListMempool) recheckTxs() {
 	mem.proxyAppConn.FlushAsync()
 }
 
+func (mem *CListMempool) GetNextTxBytes(remainBytes int64, remainGas int64, starter []byte) ([]byte, error) {
+
+	mem.updateMtx.RLock()
+	defer mem.updateMtx.RUnlock()
+
+	var lastElem *clist.CElement
+	var minPriority int64 = 0
+	if len(starter) > 0 {
+		for elem := mem.txs.Front(); elem != nil; elem = elem.Next() {
+			if bytes.Equal(elem.Value.(*mempoolTx).tx.Hash(), starter) {
```

Comment:
a second thought, we should probably just use `tx` byte to compare directly instead of using hash to avoid duplicate computation?